modules/configuring-high-availability/proc-configuring-high-availability-in-rhdh-helm-chart-deployment.adoc

```diff
@@ -0,0 +1,17 @@
+[id="proc-configuring-high-availability-in-rhdh-helm-chart-deployment"]
+= Configuring High Availability in a {product} Helm chart deployment
+
+When you are deploying {product-short} using the Helm chart, you must set `replicas` to a value greater than `1` in your custom configuration file.
+
+.Procedure
+. In your custom configuration file, set `replicas` to a value greater than `1`. For example:
```

Comment:
For single-step procedures, use an unnumbered bullet instead of a numbered list. See https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs

```suggestion
* In your custom configuration file, set `replicas` to a value greater than `1`. For example:
```

Reply:
Additionally the file name is not mentionned. Consider mentionning it for clarity.